ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.html

```diff
@@ -1,2 +1,101 @@
-<h2>Assessment content coming soon!</h2>
-<button mat-button (click)="closeSidenav()">Close</button>
+<div class="sidenav-view-assessment-container">
+  <div class="sidenav-view-assessment-header">
+    <span class="mat-h2 header-title">Assessment report</span>
+    <button class="close-button" mat-icon-button color="primary" (click)="closeSidenav()">
+      <mat-icon class="close-icon">close</mat-icon>
+    </button>
+  </div>
+  <div class="content">
+    <p>
+      Estimation for {{ srcDbType.toUpperCase() }} to for Spanner conversion:
+      {{ conversionRatePercentage.good }}% of tables can be converted automatically,
+      {{ conversionRatePercentage.ok }}% requires minimal conversion changes and
+      {{ conversionRatePercentage.bad }}% requires high complexity conversion changes.
+    </p>
+
+    <p></p>
+    <h3 class="sidenav-title">Source and destination database definitions</h3>
+    <table mat-table [dataSource]="dbDataSource" >
```

Comment:
The spacing between the header and table component should be same as the spacing between 'Conversion status by table' and the content below it : https://drive.google.com/file/d/16ACZFFcPhc4e6n3HWXnNiDIg8GURmVNQ/view?usp=share_link&resourcekey=0-gXW5_HRqxun4_M02YvquKA

Reply:
Updated